### ISSUE TITLE
Codabench: fact sheet information not available to scoring program?

### ISSUE THREAD

**Issue by johann-petrak** [codalab/codabench#1385]
It seems the fact sheet information is not available to the scoring program in any way. This is a real problem as the fact sheet can be used for lots of important information, some of which may be relevant to the scorer or just of being able to run the scorer (e.g. agreeing to understand certain conditions by checking a checkbox in the fact sheet).
Is there some way to access the fact sheet from the scorer program?

**Comment by ihsaan-ullah** [comment]
You are right, the factsheet answers are not accessible to the scoring program. If you have a situation in which you need some information for the scoring program, you can instruct the participants to submit an additional file e.g. a json file with the required information. Your scoring program can then read the settings/information from the json file and use it.

**Comment by johann-petrak** [comment]
But .... why?  Would it not easy to just provide a json file with that information in the input directory? That information surely does get collected somewhere on the server anyways, so I cannot imagine it not be trivial to place it as a json file there?
Asking to provide informaiton in a file separately when there already is a way to provide information through the web interface looks quite clumsy to me.
Also, the fact sheet can be used to double check adherence to TOS or similar which would be great to check in the scorer before actually processing a submission.

**Comment by Didayolo** [comment]
This is an interesting take. Can you describe a use case where the fact sheet answers influence the scoring program?

**Comment by johann-petrak** [comment]
The original motivation was that we have different projects for different tasks/tracks of a competition and we wanted to use the fact sheet to make sure the user has selected an option that confirms they understand which project they are submitting to. The scorer could then exit without scoring if the corrent option has not been chosen.
Another possibility could be that only registered teams get scored and the scorer compares the entered team name to a list of known team names on the server.

**Comment by Didayolo** [comment]
It is a very good suggestion. We close this issue to keep track of it in this issue:

- #1904
